modules/kinesis/src/it/resources/enrich/enrichments/javascript_script_config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "schema": "iglu:com.snowplowanalytics.snowplow/javascript_script_config/jsonschema/1-0-0",
+  "data": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "javascript_script_config",
+    "enabled": true,
+    "parameters": {
+      "script": "dmFyIGNpcGhlciA9IGphdmF4LmNyeXB0by5DaXBoZXIuZ2V0SW5zdGFuY2UoIkFFUy9DQkMvUEtDUzVQYWRkaW5nIikKdmFyIGdlbmVyYXRvciA9IGphdmF4LmNyeXB0by5LZXlHZW5lcmF0b3IuZ2V0SW5zdGFuY2UoIkFFUyIpCmdlbmVyYXRvci5pbml0KG5ldyBqYXZhLnNlY3VyaXR5LlNlY3VyZVJhbmRvbSgpKQp2YXIga2V5ID0gZ2VuZXJhdG9yLmdlbmVyYXRlS2V5KCkKY2lwaGVyLmluaXQoamF2YXguY3J5cHRvLkNpcGhlci5FTkNSWVBUX01PREUsIGtleSwgbmV3IGphdmF4LmNyeXB0by5zcGVjLkl2UGFyYW1ldGVyU3BlYyhrZXkuZ2V0RW5jb2RlZCgpKSkKCmZ1bmN0aW9uIHByb2Nlc3MoZXZlbnQpewogIHZhciBhcHBJZCA9IGV2ZW50LmdldEFwcF9pZCgpCiAgaWYgKGFwcElkID09IG51bGwpIHsKICAgIHJldHVybiBbXQogIH0gZWxzZSB7CiAgICByZXR1cm4gWyB7CiAgICAgIHNjaGVtYTogImlnbHU6Y29tLnNub3dwbG93YW5hbHl0aWNzLnNub3dwbG93L2lkZW50aWZ5L2pzb25zY2hlbWEvMS0wLTAiLAogICAgICBkYXRhOiB7IGlkOiAgbmV3IGphdmEubGFuZy5TdHJpbmcoY2lwaGVyLmRvRmluYWwoYXBwSWQuZ2V0Qnl0ZXMoKSkpIH0KICAgIH0gXQogIH0KfQ=="
```

Comment:
Agreed, but I don't know where to put that, what would you recommend ?

Reply:
For now I think the best place would be somewhere close to the JS enrichment related spec? 

Idea: you write JS code (somewhere in resources), test automatically encodes it to b64 and provides it to enrich container...would that be cool or nah? 

Or enrich could log somewhere decoded script...?

Reply:
> Idea: you write JS code (somewhere in resources), test automatically encodes it to b64 and provides it to enrich container...would that be cool or nah?

Although technically possible, it would require to write a file locally on disk, to then be able to mount it in the Docker container. That doesn't sound ideal. I'd rather wait for [this PR](https://github.com/snowplow/enrich/pull/731) to get merged. Afterwards we'll be able to pass the base64-encoded JS script as an environment variable, so it'll be possible to define it and encode it in the test directly.

> Or enrich could log somewhere decoded script...?

I would find that convenient but I'd rather not, because some customers might need to put credentials in their JS script (until we can use env vars), and we don't want to log that.

Reply:
You will like [that](https://github.com/snowplow/enrich/blob/feature/it_more_enrichments/modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Enrichments.scala#L85-L103) !

I adopted a new strategy, which is to pass all the enrichments configs as a big base64 self-describing event (schema [here](https://github.com/snowplow/enrich/blob/feature/it_more_enrichments/modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Enrichments.scala#L30)).